tests/SprykerEcoTest/Zed/Payone/Business/Payment/MethodMapper/KlarnaTest.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\Payone\Business\Payment\MethodMapper;
+
+use Generated\Shared\DataBuilder\PayoneKlarnaStartSessionRequestBuilder;
+use Generated\Shared\DataBuilder\QuoteBuilder;
+use Generated\Shared\Transfer\CurrencyTransfer;
+use Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Zed\Payone\Business\Payment\MethodMapper\Klarna;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @group Unit
+ * @group SprykerEco
+ * @group Zed
+ * @group Payone
+ * @group Business
+ * @group Payment
+ * @group MethodMapper
+ * @group KlarnaTest
+ */
+class KlarnaTest extends AbstractMethodMapperTest
+{
+    public const PAY_METHOD_TYPE = 'KIV';
+
+    protected const STANDARD_PARAMETER_CLEARING_TYPE = 'fnc';
+
+    public const AUTHORIZATION_KLARNA_REQUIRED_PARAMS = [
```

Comment:
* If the constants are not used outside the class then change modifier to the `protected`.

Reply:
++

Reply:
++